sniffy-web/src/main/java/io/sniffy/servlet/SniffyFilter.java

```diff
@@ -65,13 +65,7 @@
     public static final String HEADER_TIME_TO_FIRST_BYTE = "Sniffy-Time-To-First-Byte";
     public static final String HEADER_REQUEST_DETAILS = "Sniffy-Request-Details";
 
-    public static final String SNIFFY_URI_PREFIX =
-            "sniffy/" +
-                    Constants.MAJOR_VERSION +
-                    "." +
-                    Constants.MINOR_VERSION +
-                    "." +
-                    Constants.PATCH_VERSION;
+    public static final String SNIFFY_URI_PREFIX = "sniffy";
```

Comment:
sniffy.js is aggressively cached - we cannot remove version from JS URL otherwise it would break during updates (users will need to Ctrl+F5 in browser)
But for API calls it makes perfect sense to remove the version

Reply:
Updated. But now I should fix sniffy-ui. Right?

Reply:
I don't want to touch UI part now - will change server side to support API calls both with and without version in URL
Thanks for raising it